test/general-tests/constructor-test/check.ts

```diff
@@ -1,4 +1,33 @@
-import { hardhatStarknetCompileDeprecated, hardhatStarknetTest } from "../../utils/cli-functions";
+import path from "path";
```

Comment:
I would put these new additions to a separate test folder and call it something like cairo1-constructor-test. Or maybe even better: change the name of this cairo0 test folder to cairo0-constructor-test (and call the cairo1 test just constructor-test)

Reply:
Also did similar thing on the example repo. https://github.com/0xSpaceShard/starknet-hardhat-plugin/pull/377/commits/9b6048abf937d5c0936f1f5a4bf065acd4449935